### PR TITLE
Gm132

### PR DESCRIPTION
**Pull request by anthonyccri**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 usage() { echo "Usage: $0 [-t <tag>] [-g <geomesa.version>] [-a <accumulo.version>]" 1>&2; exit 1; }
 
 ACCUMULO_VERSION=1.7.2
-GEOMESA_VERSION=1.3.0-m3-SNAPSHOT
+GEOMESA_VERSION=1.3.1-SNAPSHOT
```
Comment (jnh5y) [inline build.sh:6]: Should the GM version be 1.3.2-SNAPSHOT?

Comment (anthonyccri) [reply build.sh:6]: Eh, it is an arg in the ./build.sh script.

File: geodocker-geoserver/Dockerfile.template [hunk 2]
```diff
@@ -67,7 +67,7 @@ RUN set -x \
 # Install GeoMesa Process Jar
 RUN set -x \
     && cd /opt/tomcat/webapps/geoserver/WEB-INF/lib/ \
-    && cp /opt/geomesa/dist/gs-plugins/geomesa-process_2.11-${GEOMESA_VERSION}.jar .
+    && cp /opt/geomesa/dist/gs-plugins/geomesa-process-wps_2.11-${GEOMESA_VERSION}.jar .
```
Comment (jnh5y) [inline geodocker-geoserver/Dockerfile.template:70]: @aheyne Is this the right name for the geomesa-process-wps jar in 1.3.2?

Comment (aheyne) [reply geodocker-geoserver/Dockerfile.template:70]: Yep,` geomesa-process-wps_2.11-1.3.2-SNAPSHOT.jar`

File: build.sh (hunk 1 above)
Comment (jnh5y) [reply build.sh:6]: Can we bump this to 1.3.2-SNAPSHOT?  That let us default to the current latest?

File: geodocker-base/Dockerfile.template [hunk 3]
```diff
@@ -10,11 +10,11 @@ ARG ACCUMULO_VERSION
 ARG THRIFT_VERSION
 
 # JAVA
```
Comment (gbrionesCCRI) [inline geodocker-base/Dockerfile.template:12]: I submit a PR with some code from my fork for this template, should help with updating the java version in future updates

**Pull request merged**